### DISCUSSION TITLE
Delete Item lambda handler

### DISCUSSION THREAD

**Discussion by Russell500** [gp-seattle/rms-app#89] [Backend Questions] [answered]
I have been getting this error and do not know how to fix it unless I change the name to void
![image](https://user-images.githubusercontent.com/87156867/127203729-0bf7dada-e978-42a2-9af9-1a40db477878.png)

**Comment by yauj** [comment, marked as answer]
@Russell500 Can you try and play around with the location of the `.then` that is giving the error? Like I think you can try move it up to the end of line 65 for example.

**Reply by Russell500** [reply]
I can try that

**Reply by Russell500** [reply]
@yauj moving it to 66 and 65 did not work it instead gave different errors

**Reply by Russell500** [reply]
fixed
